diesel/src/result.rs

```diff
@@ -124,6 +124,9 @@ pub enum DatabaseErrorKind {
     /// A check constraint was violated.
     CheckViolation,
 
+    /// The connection to the server was unexpectedly closed.
```

Comment:
I would like to have a comment here stating that we emit this error message on best effort basis, so that it is clear that we may miss situations where this occurs.

Reply:
Added. Thanks for your review!